frontend/websites/puconvocation-com/src/app/[locale]/(console)/console/attendees/page.tsx

```diff
@@ -10,15 +10,167 @@
+                className="bg-red-600 text-white hover:bg-red-700"
+              >
+                {showAttendees ? "Unlock" : "Lock"}
+              </Button>
+            </CardContent>
+          </Card>
+
+          {/* Attendee List */}
+          <Card className="h-fit w-full border border-gray-300 p-4 shadow-none">
+            <CardHeader>
+              <CardTitle>Attendee List</CardTitle>
+              <CardDescription>
+                Showing detailed list of registered attendees with their
+                respective information.
+              </CardDescription>
+            </CardHeader>
+            <CardContent className="flex h-96 flex-col overflow-y-auto">
+              {isAttendeeLoading ? (
+                <div className="flex h-full items-center justify-center">
+                  <ProgressBar type="circular" />
+                </div>
+              ) : attendeeError ? (
+                <p className="text-red-600">Error loading attendees</p>
+              ) : currentAttendees.length > 0 ? (
+                <table className="min-w-full table-auto border-collapse">
+                  <thead>
+                    <tr>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Convocation Id
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Name
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Enclosure
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Row
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Seat
+                      </th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {currentAttendees.map((a) => (
+                      <tr
+                        key={a.convocationId}
+                        className="border-b text-center transition-colors duration-200 hover:bg-gray-100"
+                      >
+                        <td className="px-4 py-2">{a.convocationId}</td>
+                        <td className="px-4 py-2">{a.studentName}</td>
+                        <td className="px-4 py-2">{a.allocation.enclosure}</td>
+                        <td className="px-4 py-2">{a.allocation.row}</td>
+                        <td className="px-4 py-2">{a.allocation.seat}</td>
+                      </tr>
+                    ))}
+                  </tbody>
+                </table>
+              ) : (
+                <p>No attendees found</p>
+              )}
+            </CardContent>
+          </Card>
+        </div>
+
+        {/* Pagination Controls */}
+        <div className="mt-4 flex w-full justify-end">
```

Comment:
The pagination must be incorporated with the table itself and not as separate block outside the table.

Also the pagination must also show the total pages along with current page.
For example: `2/100`